lib/cc/analyzer/container.rb

```diff
@@ -57,10 +57,10 @@ def run(options = [])
         _, status = Process.waitpid2(pid)
 
         if @timed_out
-          duration = timeout
+          duration = timeout * 1000
           @listener.timed_out(container_data(duration: duration))
         else
-          sleep 0.01 until !t_out.alive? && !t_err.alive?
+          sleep 1 until !t_out.alive? && !t_err.alive?
```

Comment:
Comment lost to the diff:

I wondered this last time it came though: doesn't this basically just re-implement Thread#join?